scripts/generate_calibs_gen3.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+# This file is part of ap_verify_hits2015.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Script for automatically generating calibs for this dataset. It takes roughly
+# 6 hours to run on lsst-devl.
+# Running this script allows for calibs to incorporate pipeline improvements.
+# It makes no attempt to update the set of input exposures or their validity
+# ranges; they are hard-coded into the file.
```

Comment:
This is a nice little script version of my tedious calib creation process, thank you!

Can you please add an example for how to run it from the command line (with typical options)?

Reply:
How is this:
```suggestion
# Script for automatically generating calibs for this dataset. It takes roughly
# 6 hours to run on lsst-devl.
# Running this script allows for calibs to incorporate pipeline improvements.
# It makes no attempt to update the set of input exposures or their validity
# ranges; they are hard-coded into the file.
#
# Example:
# > nohup generate_calibs_gen3.sh -c "u/me/DM-123456" &
# produces certified calibs in /repo/main in the u/me/DM-123456-calib collection.
# See generate_calibs_gen3.sh -h for more options.
```